.github/workflows/contracts-e2e-tests-and-deploy.yaml

```diff
@@ -279,13 +290,23 @@ jobs:
       - name: Cache contracts' target directories in S3 bucket
```

Comment:
but why this step is even here? There is already caching, using our own [cache action](https://github.com/Cardinal-Cryptography/aleph-node/pull/714/files#diff-64b660450dbf52b1f9f9bad88cc7ac7817e727c15d40c74b0738f9f8011bbce4R187)?

Reply:
I have removed caching targets to S3